### PR TITLE
added the link to solution of Project-2 

### PR DESCRIPTION
**Pull request by ChaturvediSulabh**
This is a PR template. If you are adding a solution link to the curriculum, leave this as is. If not, delete it and write the message you wish.

Thank you,

The Odin Project team.

### CODE REVIEW

**Comment by 105ron** [comment]
Hi @ChaturvediSulabh,
This repository holds the files that make up our website. We require you to add a link to your solution, not the actual solution. If you need help with how to do this, head on over to the general room on our Discord chat room. Thank you.

File: ruby_programming/basic_ruby/project_building_blocks.md [hunk 1]
```diff
@@ -18,1316 +19,1321 @@ There's a video about it [from Harvard's CS50 class](https://www.youtube.com/wat
...
+- [Dominik Stodolny's solution](https://github.com/dstodolny/ruby_building_blocks/blob/master/lib/caesar_cipher.rb)
+- [Andy Linteau's solution](https://github.com/linteau/ceasar_cipher/blob/master/cipher.rb)
+- [Josh Naughton's solution](https://github.com/ThothLogos/odin-rubyprogramming/blob/master/01_caesar_cipher.rb)
+- [Jason Matthews's solution](https://github.com/fo0man/ruby_blocks/blob/master/caeser_cipher.rb)
+- [Kevin Weir's solution](https://github.com/IDCrisis2/the_odin_project/blob/master/Ruby%20Building%20Blocks/caesar_cipher.rb)
+- [chasmani's solution](https://github.com/chasmani/RubyBuildingBlocks/blob/master/caesar_cipher.rb)
+- [Kevin Mulhern's solution](https://github.com/KevinMulhern/ruby_building_blocks/blob/master/caesar_cipher.rb)
+- [Emil Johansen's solution](https://github.com/EmilOJ/odin_project/blob/master/Ruby_Building_Blocks/caesar_cipher.rb)
+- [Sasikala's solution](https://github.com/Sasikala-Ravichandran/my_ruby_projects/blob/master/caesar_cipher.rb)
+- [Sam Padrul's solution](https://github.com/sampadrul/Ruby-Projects/blob/master/caesar%20cipher/caesa
```
Comment (ChaturvediSulabh) [inline ruby_programming/basic_ruby/project_building_blocks.md]: I have added the link to my solution

File: ruby_programming/basic_ruby/project_building_blocks.md [hunk 2]
```diff
@@ -18,1316 +19,1321 @@ There's a video about it [from Harvard's CS50 class](https://www.youtube.com/wat
...
-* [Anna Olak's Solution](https://github.com/anna1901/ruby_building_blocks_exercises/blob/master/caesar_cipher.rb)
-* [David Watkin's Solution](https://github.com/DavidWatkinCode/the_odin_project/blob/master/ruby/ruby_programming/caesar_cipher.rb)
-* [Rafael Rodriguez Garcia's Solution](https://github.com/rrg1459/caeser)
-* [Robert Suazo's Solution](https://github.com/rsuazo/ruby_exercises/tree/master/caesar)
-* [DalandanJuice's Solution](https://github.com/DalandanJuice/caesar-cipher/blob/master/caesar-cipher.rb)
-</details>
+- Add your solution below this line!
+- [Awais Ali Khan](https://github.com/awaisalikhan/ruby_programming/blob/master/caesar_cipher.rb)
+- [Mike Farias' Solution](https://github.com/mkezmkez/ruby-buildingblocks/blob/master/practice1.rb.arb)
+- [Sulabh Chaturvedi's Solution](https://github.com/ChaturvediSulabh/the-odin-project/blob/master/Building_Blocks/project1/caeser_cipher.rb)
```
Comment (ChaturvediSulabh) [inline ruby_programming/basic_ruby/project_building_blocks.md]: this is the link to my solution

**Comment by ChaturvediSulabh** [comment]
> Hi @ChaturvediSulabh,
> This repository holds the files that make up our website. We require you to add a link to your solution, not the actual solution. If you need help with how to do this, head on over to the general room on our Discord chat room. Thank you.

I apologize for the misleading PR title, I have added the link to my solution and not the solution 

> Hi @ChaturvediSulabh,
> This repository holds the files that make up our website. We require you to add a link to your solution, not the actual solution. If you need help with how to do this, head on over to the general room on our Discord chat room. Thank you.

apology for the misleading pr title. though, I've added  [my solution link](https://github.com/TheOdinProject/curriculum/pull/13173/files/e861d689ae22541bea3c24bec1fc0d19d956e294#r347106581) and not the actual solution.

**Comment by 105ron** [comment]
Ok, it looks like you have an auto formatter enabled. There's  2,532 changes in the file and that is too many for me to review in a single PR. Could you disable it and only change the line you need to add your solution?

**Pull request closed without merging**